CMakeLists.txt

```
@@ -40,3 +40,4 @@ endif ()
 
 add_subdirectory(lib)
 add_subdirectory(example-app)
+add_subdirectory(tests)
```

I think you should add an option in `cmake/options.cmake` that would look like this:
```cmake
option(${PROJECT_PREFIX}_ENABLE_TESTING "Enable Testing" OFF)
```
We should turn it off by default (because most people don't care about tests).  
You can then turn it on with the `cmake -DVULK_ENABLE_TESTING=ON [build_folder]` command.

Then the code here would look like this:
```suggestion
if (${PROJECT_PREFIX}_ENABLE_TESTING)
    add_subdirectory(tests)
endif ()
```